prov/rxm/src/rxm_attr.c

```diff
@@ -57,6 +59,7 @@ struct fi_ep_attr rxm_ep_attr = {
 };
 
 struct fi_domain_attr rxm_domain_attr = {
+	.name = rxm_prov_name,
```

Comment:
This should be dropped. Domain name would be same as the underlying core provider. See #2808